src/snowflake/snowpark/session.py

```diff
@@ -978,6 +980,7 @@ def table_function(
 
         See Also:
             - :meth:`DataFrame.join_table_function`, which lateral joins an existing :class:`DataFrame` and a SQL function.
+            - :meth:`Session.generator`, which is used to instantiate a :class:`DataFrame` using Generator table function.
```

Comment:
Let's explicitly mention that this method is not working with `generator` table function (or even you can raise a ValueError in code to remind the user to use Session.generator)

Reply:
Raising error might be difficult in case users create UDTF with name set as "generator". I will update docstring to explicitly say that `Session.table_function` does not support generator table function